src/styles/record-manager.css

```diff
@@ -237,7 +249,7 @@ td.actions button {
 
 @media screen and (min-width: 576px) and (max-width: 992px) {
     .container {
-        width: auto;
+        width: 100%;
```

Comment:
Agree, it is better to leave as it is, I set **width** to 100% before, because I had a problem with the container items being shrinked in width(overly), but now I left it as **auto**, as it doesn't break anything anymore.

Done: changed it to the **auto** again and squashed